src/Menu/Menu.jsx

```diff
@@ -35,6 +33,17 @@ MenuContent.defaultProps = {
   className: null,
 };
 
+const menuPropTypes = {
```

Comment:
There was an eslint warning after upgrading edx/frontend-build that was complaining about reading propTypes from the `Menu` component since there's no guarantee propTypes will remain in a production build (there are plugins that remove propTypes from production bundles).

Moving the propTypes to their own variable gets around this issue.